lib/ansible/module_utils/dimensiondata.py

```diff
@@ -0,0 +1,393 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Dimension Data
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#   - Aimon Bustardo <aimon.bustardo@dimensiondata.com>
+#   - Mark Maglana   <mmaglana@gmail.com>
+#
+# Common methods to be used by versious module components
+import os
+import ConfigParser
```

Comment:
Sorry, just checking - do you mean:

```python
from ansible.module_utis.six.moves import configparser
```
or
```python
from ansible.module_utis.six.moves import ConfigParser
```

(I've never seen this code before so it's a lot to take in, but it looks like the latter rather than the former?)